api/resolvers/item.js

```diff
@@ -299,6 +299,12 @@ function typeClause (type) {
     case 'bios':
       return ['"Item".bio = true', '"Item"."parentId" IS NULL']
     case 'bounties':
+      // if (subType === 'paid') {
+      //   return ['"Item".bounty IS NOT NULL', '"Item".bountyPaidTo IS NOT NULL', '"Item"."parentId" IS NULL']
+      // } else if (subType === 'unpaid') {
+      //   return ['"Item".bounty IS NOT NULL', '"Item".bountyPaidTo IS NULL', '"Item"."parentId" IS NULL']
+      // }
+      
```

Comment:
@huumn Is `bountyPaidTo` a field because I tried the above but it returns an error. Is this "the way" to know whether a bounty has been paid?

Reply:
You need to use quotation marks for case-sensitive SQL fields (see surrounding code)